GameOfLifeApp/Serializer.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace GameOfLifeApp
+{
+    /// <summary>
+    /// Saves JSON data to a file.    
+    /// </summary>
+    public class Serializer
+    {
+        public GameLogic? data;
+        //public Display? display;
+
+        public string fileName = @"C:\Users\a.vlasova\source\repos\AnitaRiga\GameOfLifeApp\GameData.json";
```

Comment:
so this means that only you on your computer can read or save the file and only if it does not change the location, this is not a correct approach, google how to dynamically find the folder where your application is and save it there.

Reply:
Using Environment.CurrentDirectory to get a path of the current working directory.